README.md

```diff
@@ -0,0 +1,206 @@
+<a name="readme-top"></a>
```

Comment:
- [ ] Good job adding tests!  Please add a "tests" section on your readme stating the correct way to run your tests.

_tip:  The 'rspec' command alone won't do:_

![image](https://github.com/MarcoAurelioAntonio/Global-Academy-Server/assets/57421823/1b56d639-f18f-45cc-b182-cf6c43766670)

This will work:

rspec ./spec/models

![image](https://github.com/MarcoAurelioAntonio/Global-Academy-Server/assets/57421823/1f7a32ec-39e0-438c-8875-ed56e8be8b1e)